### DISCUSSION TITLE
import command not working

### DISCUSSION THREAD

**Discussion by wolfy1731** [Rapptz/discord.py#7178] [Q&A]
I have tried everything to get discord.py installed but the pip install doesnt work then the import isnt working as it says there is no bash or command named import

**Comment by jelni** [comment]
To install discord.py type
```sh
pip install discord.py
```
in your terminal. Then in your Python file you can type
```py
import discord
```
Also remember, this library is not good for beginners in Python or programming in general.

**Comment by mesub7** [comment]
Make sure that you're using import in a file or the python REPL (it's likely to be the file though).
